### DISCUSSION TITLE
[Feature] EXTERNAL LIBRARY imported/migrated under new template style

### DISCUSSION THREAD

**Discussion by Dbabby-lab** [immich-app/immich#14714] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I found parallel requests but not quite the identical use-case that I'm looking for.  Apologies if it exists and I haven't found it.  

I've been using Immich for a couple of months and I adore it.  It's exactly what I was looking for.  Before using Immich, my backups for all of our pics has been sloppy.  I had many backups in lots of places because I feared losing photos.   When I used the Google Takeout process to create a full external backup of all of our past 20 years of media, I ended up with MANY duplicates, unnecessary pics and videos, wrongly-dated-photos, etc...

So when I started using Immich, I took all our past photos and made them an external library, and started our phones loading new photos into Immich.  This has worked great.  But even though I've weeded out tens of thousands of duplicates (which include 200gb of media), those all still exist in the external library.  I have spent many hours updating the timestamps of wrong photos, deleting duplicates, and whittling down our library so that what's currently in Immich are the photos I want.  

What I would love is a "job" which would move all of my current assets into my true immich library, thereby rendering the external library obsolete.  In my particular use-case, I would love if it would re-name and re-organize all of those files.  The result would be a clean library that would render my previous mess obsolete.  As it is, the mess persists.  

This feature would be a God-send! 

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1699
